reco_utils/dataset/movielens.py

```diff
@@ -415,6 +456,45 @@ def _load_datafile(size, local_cache_path):
 
     return datapath, item_datapath
 
+def download_datafile(size, local_cache_path='./data/ml.zip'):
```

Comment:
ok, so here you you want to just download ML right? but this code is very similar to some code that is inside `_load_datafile(size, local_cache_path)`. We should split this function and would do [single responsibility](https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines#single-responsibility). One suggestion is:
1) do one function called download_movielens
2) another that extract the content
3) refactor _load_datafile to only load the datafile

@loomlike can help on this, since we did the movielens loader

Reply:
Please check #647 for details about downloading movielens zip file and extracting. Basically, under discussion about having separate download and extract functions.

Reply:
The movielens dataloader changes are merged into staging. Now you can use Download and Extract functions as well as directly load into DataFrame as usual.

Reply:
Cool. Thank you! I changed it to not having to use download function as you suggested